ingestion/src/metadata/ingestion/source/trino.py

```diff
@@ -8,78 +8,49 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-"""Trino source module"""
 
 import logging
 import sys
-from typing import Iterable
+from typing import Optional
 from urllib.parse import quote_plus
 
 import click
-from sqlalchemy.inspection import inspect
 
 from metadata.generated.schema.entity.services.databaseService import (
     DatabaseServiceType,
 )
-from metadata.ingestion.models.ometa_table_db import OMetaDatabaseAndTable
 from metadata.ingestion.ometa.openmetadata_rest import MetadataServerConfig
 from metadata.ingestion.source.sql_source import SQLSource
 from metadata.ingestion.source.sql_source_common import SQLConnectionConfig
 
 logger = logging.getLogger(__name__)
 
-
 class TrinoConfig(SQLConnectionConfig):
-    """Trinio config class -- extends SQLConnectionConfig class
-
-    Attributes:
-        host_port:
-        scheme:
-        service_type:
-        catalog:
-        database:
-    """
-
     host_port = "localhost:8080"
     scheme = "trino"
     service_type = DatabaseServiceType.Trino.value
     catalog: str
-    database: str
+    include_views = False
+    params: Optional[dict] = None
 
     def get_connection_url(self):
         url = f"{self.scheme}://"
-        if self.username is not None:
-            url += f"{self.username}"
-            if self.password is not None:
+        if self.username:
+            url += f"{quote_plus(self.username)}"
+            if self.password:
                 url += f":{quote_plus(self.password.get_secret_value())}"
             url += "@"
         url += f"{self.host_port}"
-        if self.catalog is not None:
-            url += f"/{self.catalog}"
-            if self.database is not None:
-                url += f"/{self.database}"
-
-        if self.options is not None:
-            if self.database is None:
-                url += "/"
+        url += f"/{self.catalog}"
+        if self.params is not None:
             params = "&".join(
-                f"{key}={quote_plus(value)}"
-                for (key, value) in self.options.items()
-                if value
+                f"{key}={quote_plus(value)}" for (key, value) in self.params.items() if value
```

Comment:
`options` is specific to SQLAlchemy. `params` is specific to `Trino` and `Presto`.